rpcs3/rpcs3qt.vcxproj.user

```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup />
+</Project>
```

Comment:
this file already exist, it is rpcs3.vcxproj.user ^^, i fix this problem in my pr